### PR TITLE
Make the Changelog reachable from the Landing Page

### LINKED ISSUE #1463: Make the Changelog reachable from the Landing Page
Note: this has been selected as an Outreachy issue. To be assigned, you must have completed the instructions in https://github.com/ocaml/ocaml.org/issues/1553.

# Task

Currently, the only way to reach the https://ocaml.org/changelog page is to through the link in the footer. There are no banners or navigation that lead there.

On the main landing page at https://ocaml.org/, we currently have a "Releases" section. This appears a natural place to link to the changelog:

Screenshot 2023-10-03 at 16-51-50 Welcome to a World of OCaml

Here is a design to make the changelog reachable from the main landing page:
Screenshot from 2023-10-03 16-49-50

A corresponding Figma document is available here: https://www.figma.com/file/fCs065Xt968dFPDVXGJkM9/changelog-section?type=design&node-id=0-1&mode=design&t=FBKltgYF79AP9cRx-0

You will have to do this:

1. modify the template `home.eml` to take a list of changelog entries as a parameter
2. modify the template `home.eml` to render changelog component as given in the Figma document next to the releases. This requires some Tailwind skills to style the HTML you're rendering.
3. In `handler.ml` pass the first three changelog entries to the template.

### PR DESCRIPTION
**Pull request by tinniaru3005**
### Tasks
- Create a section to display CHangelogs

### Screenshots
changelog

Closes #1463

### CODE REVIEW

File: src/ocamlorg_frontend/pages/home.eml [hunk 1]
```diff
@@ -263,24 +263,50 @@ Layout.render
...
-        <div class="text-primary-600 text-sm mb-4 uppercase">Recent Release</div>
-        <h3 class="font-bold mb-6">
-          <%s latest_release.version %> <span class="text-sm font-medium">(<%s latest_release.date %>)</span>
-        </h3>
-      </a>
-      <div class="prose prose-sm text-white"><%s! latest_release.highlights_html %></div>
+
+<div class="flex gap-10 lg:gap-15 flex-col lg:flex-row justify-between items-center">
+  <div class="overflow-hidden lg:flex-1 mt-10 lg:mt-0 flex items-center">
+    <div class="container-fluid flex flex-col items-left">
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/home.eml]: `container-fluid` needs to wrap both of the columns

File: src/ocamlorg_frontend/pages/home.eml [hunk 2]
```diff
@@ -263,24 +263,50 @@ Layout.render
...
-    <div class="bg-pattern p-6 mt-8 rounded-xl w-full lg:w-2/3 text-white">
-      <a href="<%s Url.release latest_release.version %>">
-        <div class="text-primary-600 text-sm mb-4 uppercase">Recent Release</div>
-        <h3 class="font-bold mb-6">
-          <%s latest_release.version %> <span class="text-sm font-medium">(<%s latest_release.date %>)</span>
-        </h3>
-      </a>
-      <div class="prose prose-sm text-white"><%s! latest_release.highlights_html %></div>
+
+<div class="flex gap-10 lg:gap-15 flex-col lg:flex-row justify-between items-center">
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/home.eml]: I think we will also need some bottom padding so the cards don't touch the next section.

**Review by sabine — COMMENTED** [review]
This is looking really nice already @tinniaru3005! :+1:  The next steps here are to

1. add another parameter `changelog` to the `home.eml` template
2. get changelog entries in the `home` function in `handler.ml` and pass them to the `Ocamlorg_frontend.home` template.
3. Render the changelog entries in the right-hand side column

**Comment by tinniaru3005** [comment]
Thanks a lot @sabine for the guidance. I am working on the changes. Will keep you updated.

**Comment by sabine** [comment]
@tinniaru3005 what is the status on this? Are you interested in continuing to work on this PR? If yes, I'll wait. If no, I'll free the issue for someone else to work on it.

**Comment by tinniaru3005** [comment]
Yes @sabine I am working on it. Will update by today. The work was delayed due to my health issues. Thanks for your patience.

**Comment by sabine** [comment]
@tinniaru3005 Sorry about that, hope you'll feel better soon. And no worries, I'm just checking on all the open PRs. :+1:

**Comment by tinniaru3005** [comment]
> @tinniaru3005 Sorry about that, hope you'll feel better soon. And no worries, I'm just checking on all the open PRs. 👍

Yes I am fine now. Was released from hospital on 1st. Now I am fine and back to normal work.

**Comment by tinniaru3005** [comment]
> This is looking really nice already @tinniaru3005! 👍 The next steps here are to
> 
> 1. add another parameter `changelog` to the `home.eml` template
> 2. get changelog entries in the `home` function in `handler.ml` and pass them to the `Ocamlorg_frontend.home` template.
> 3. Render the changelog entries in the right-hand side column

Hey @sabine can you please provide more details on where exactly I have to make the changes for the steps 1 and 2. I tried referring to the `changelog.eml` but got errors in rendering the items. I am doubtful on where in the `home.eml` template I have to add the `changelog` parameter.

**Comment by sabine** [comment]
Hey @tinniaru3005, if you commit your current state to the branch, I can take a look at the code. It is very hard for me to guess where the problem is. Thanks!

**Pull request closed without merging**